### ISSUE TITLE
Logitech G Hub uninstallation does not detect leftover files

### ISSUE THREAD

**Issue by rk-exxec** [Klocman/Bulk-Crap-Uninstaller#445]
I uninstalled the Logitech G Hub application. 
In this step to remove leftover files however there were no entries, despite the userdata folder  `Appdata\Local\LGHUB` still being there. 
I assume that this is because the foldername is not identical to the program name.

The program installs in `C:\Program Files\LGHUHB`. 
Userdata is in `Appdata\Local\LGHUB`. The userdata was not removed 

System: Win 10 Pro 22H2

**Comment by Klocman** [comment]
Yes, the folder name was most likely too different. You can sometimes still see these items if you turn on showing low confidence items, but the junk window won't show up if there is nothing of good confidence to show.

**Comment by Klocman** [comment]
I gave this some thought and there's no easy fix, at least without hardcoding this folder name.
It could be caught by adding more in depth comparisons like splitting all words and trying all combinations of initials and full words, but that would slow down detection speed by quite a bit while still giving only low confidence.

**Comment by rk-exxec** [comment]
Can you get the install folder name during the uninstall process?
In this case you could search for folders with the same name as the main install folder in the AppData dirs.

**Comment by Klocman** [comment]
It does take install folder into account. I'm guessing the confidence is just too low to show up since very short folder names are penalized.

**Comment by gursuj** [comment]
Wanted to add that uninstalling WordPress Studio also doesn't detect its leftover `~/.studio` folder
